Index: src/main/kotlin/com/sfinancial/permission/userpermission/UserPermission.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.sfinancial.permission.userpermission\r\n\r\nimport com.sfinancial.Account.AccountInterface\r\nimport com.sfinancial.Account.UserAccount\r\nimport com.sfinancial.group.GroupInterface\r\nimport com.sfinancial.login.LoginInterface\r\nimport com.sfinancial.verifier.VerifierUser\r\n\r\nclass UserPermission(\r\n        private val groupInterface: GroupInterface? = null,\r\n        private val accountInterface: AccountInterface? = null\r\n): UserPermissionInterface {\r\n\r\n    override fun registerAccount(){\r\n        try {\r\n            if (VerifierUser(groupInterface).verifier()){\r\n            }\r\n        }\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/kotlin/com/sfinancial/permission/userpermission/UserPermission.kt	(revision d91ee64f4ca9c1eeaf14ef3d0501832f6f3ef786)
+++ src/main/kotlin/com/sfinancial/permission/userpermission/UserPermission.kt	(date 1604336278920)
@@ -2,18 +2,20 @@
 
 import com.sfinancial.Account.AccountInterface
 import com.sfinancial.Account.UserAccount
+import com.sfinancial.database.DBInterface
 import com.sfinancial.group.GroupInterface
 import com.sfinancial.login.LoginInterface
 import com.sfinancial.verifier.VerifierUser
 
 class UserPermission(
         private val groupInterface: GroupInterface? = null,
-        private val accountInterface: AccountInterface? = null
+        private val accountInterface: AccountInterface? = null,
+
 ): UserPermissionInterface {
 
     override fun registerAccount(){
         try {
-            if (VerifierUser(groupInterface).verifier()){
+            if (VerifierUser(groupInterface).verifier()) {
             }
         }
     }
diff --git gradlew gradlew
